src/integrationtest/java/com/ericsson/ei/frontend/CommonSteps.java

```diff
@@ -0,0 +1,155 @@
+package com.ericsson.ei.frontend;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.tomcat.util.codec.binary.Base64;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Ignore;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.test.context.SpringBootContextLoader;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.support.AbstractTestExecutionListener;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+
+import com.ericsson.ei.utils.AMQPCommunication;
+import com.ericsson.ei.utils.HttpRequest;
+import com.ericsson.ei.utils.HttpRequest.HttpMethod;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+@Ignore
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = EIFrontendApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT)
+@ContextConfiguration(classes = EIFrontendApplication.class, loader = SpringBootContextLoader.class)
+@TestExecutionListeners(listeners = { DependencyInjectionTestExecutionListener.class, CommonSteps.class })
+public class CommonSteps extends AbstractTestExecutionListener {
+
+    @LocalServerPort
+    private int frontendPort;
+    private HttpRequest httpRequest;
+    private ResponseEntity<String> response;
+
+    private static final String HOST = "localhost";
```

Comment:
If we are going ise these test in a system tests as well, these parameters need to be configurable.

Reply:
I guess you could simply modify it when the time comes. Getting host for every component used. The rabbit user and password must be set in docker-compose file in backend I guess so that you both set the credentials when rabbit goes up and get them in the test with env variable.